share/spack/gitlab/cloud_pipelines/configs/darwin/aarch64/config.yaml

```diff
@@ -1,3 +1,5 @@
 config:
+  build_stage:
```

Comment:
I would be curious if the previous issue is now fixed by using GoPackage for fzf. I am okay to experiment with that later, but I know this can cause issues with llvm built clang where it injects expired secrets into the version info.

Reply:
fzf is fixed now, yes. I think the original problem was larger than fzf though, although fzf would have contributed to it. I've removed this `build_stage` setting now, reverting to default setting. I'll monitor and we can see if the problem comes back.